hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/BaseHoodieWriteClient.java

```diff
@@ -1637,8 +1637,6 @@ protected void setWriteSchemaForDeletes(HoodieTableMetaClient metaClient) {
         } else {
           throw new HoodieIOException("Latest commit does not have any schema in commit metadata");
         }
-      } else {
-        throw new HoodieIOException("Deletes issued without any prior commits");
```

Comment:
Instead of do nothing, can we log some info here.

Reply:
What is the log content?

Reply:
> Instead of do nothing, can we log some info here.

log.warn("There is no instance data in the current table"); ??

Reply:
Yeah, let's leave some loggings here.

Reply:
I've already done that. Any other questions so far?